include/vapi/vsl_int.h

```diff
@@ -91,6 +91,7 @@ enum VSL_tag_e {
 	SLT__Bogus = 0,
 #define SLTM(foo,flags,sdesc,ldesc)	SLT_##foo,
 #include "tbl/vsl_tags.h"
+	SLT__VXID = 253, /* XXX: does it belong after SLT__Reserved? */
```

Comment:
I suspected this kind of answer, that's why I asked in the first place. I will try a new patch, thanks.